users/api/http/transport.go

```diff
@@ -47,19 +50,47 @@ func MakeHandler(svc users.Service, tracer opentracing.Tracer, l log.Logger) htt
 	))
 
 	mux.Get("/users", kithttp.NewServer(
-		kitot.TraceServer(tracer, "register")(userInfoEndpoint(svc)),
+		kitot.TraceServer(tracer, "user_info")(userInfoEndpoint(svc)),
 		decodeViewInfo,
 		encodeResponse,
 		opts...,
 	))
 
+	mux.Post("/password/reset-request", kithttp.NewServer(
+		kitot.TraceServer(tracer, "res-req")(passwordResetRequestEndpoint(svc)),
+		decodePasswordResetRequest,
+		encodeResponse,
+		opts...,
+	))
+
+	mux.Put("/password/reset", kithttp.NewServer(
```

Comment:
This here looks like hard-coded - do we use something that was configures in `main`, and if not why is this configurable in `main`?

Reply:
this is the flow
1. User opens a page for password reset in a browser where he enters email and clicks submit ( that request goes as POST to /password/reset-request )
2. User gets a link in email, link is with token, clicks on a link making a GET request and lands on UI page where he can enter new and confirm password ( url for creating link to this page is configurable since it may be convenient depending on the front being used )
3. User clicks submit in previously opened page and then makes a PUT request to `/password/reset` and if everything is OK password is changed.